packages/dapp/src/store/localStorage.ts

```diff
@@ -18,6 +18,7 @@ export const storageConnectorConfig: LocalStorageConnectorConfig = {
     'bootstrapped',
     'lodgingFacilities',
     'spaces',
```

Comment:
In my opinion, it is better to re-name search-related state variables to something like searchSpaces and searchTimestamp. Our application will have a lot of other functionality. For example - management of facilities and spaces. "spaces" more fits into that feature than for search